app/src/main/java/com/sildev/tvshows/data/repository/TVShowRepository.kt

```diff
@@ -0,0 +1,25 @@
+package com.sildev.tvshows.data.repository
+
+import com.sildev.tvshows.data.model.TVShow
+import com.sildev.tvshows.data.repository.source.TVShowDataSource
+import com.sildev.tvshows.data.repository.source.remote.OnResultListener
+
+class TVShowRepository constructor(
+    private val remote: TVShowDataSource.Remote, private val local: TVShowDataSource.Local
+) : TVShowDataSource.Remote, TVShowDataSource.Local {
+    override fun getTVShows(listener: OnResultListener<MutableList<TVShow>>) {
```

Comment:
chú ý lần cách 1 dòng ra nhé 
```suggestion
) : TVShowDataSource.Remote, TVShowDataSource.Local {

    override fun getTVShows(listener: OnResultListener<MutableList<TVShow>>) {
```